code/modules/power/tesla/coil.dm

```diff
@@ -12,6 +12,11 @@
 	var/last_zap = 0
 	var/datum/wires/tesla_coil/wires = null
 
+	// Executing a traitor caught releasing tesla was never this fun!
+	can_buckle = TRUE
```

Comment:
How exactly are the meant to get buckled to something that's density = 1? You can't move people on top of it, it acts as a wall. If the answer is 'check_loc = FALSE', my testing suggests that doesn't work.

Reply:
@Kyep any suggestion on how to fix.

Reply:
Well, (1) is intended. (4) seems fine. (2) and (3) seem odd. It seems like it should check for 'grabbed' not 'pulled'.